src/main/java/it/gov/pagopa/payhub/auth/utils/AuditUtils.java

```diff
@@ -0,0 +1,41 @@
+package it.gov.pagopa.payhub.auth.utils;
+
+import it.gov.pagopa.payhub.auth.dto.AuditLogDTO;
+import java.time.Instant;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class AuditUtils {
+  private AuditUtils() {}
+
+  private static final String VENDOR = "PagoPa";
```

Comment:
This should be generic :)
```suggestion
  private static final String VENDOR = "PiattaformaUnitaria";
```